api/middlewares/errorHandler.js

```diff
@@ -1,7 +1,12 @@
 const errorHandlerMiddleware = (error, req, res, next) => {
   res
     .status(error.status)
-    .json({ name: error.name, status: error.status, message: error.message });
+    .json({
+      name: error.name,
+      status: error.status,
+      message: error.message,
+      type: error.type,
```

Comment:
```
{
 ...error,
}
```
Contract of error is look like that `{ name, status, message }`

Reply:
Sorry that i did't inform you but we really need this field for error validation on frontend part

Reply:
You can just use a new status code to manage this error

Reply:
Done